visualization/app/codeCharta/ui/codeMap/rendering/codeMapVertexShader.glsl

```diff
@@ -6,6 +6,7 @@ attribute vec3 defaultColor;
 attribute vec3 deltaColor;
 attribute highp float subGeomIdx;
 attribute highp float delta;
+attribute lowp float isHeight;
```

Comment:
Where does it come from? Is it a boolean?

Reply:
This attribute is sent from the geometry building function. It is a float number, as no Boolean attributes can be sent to the shaders.

Reply:
Thanks for the explenation. I thought that isHeight is a `number[]` coming from `isTopSide(x)`. Multiplying it with a scalar wouldn't make much sense.